scripts/available_software/available_software.py

```diff
@@ -35,11 +34,7 @@
 
 def main():
     os.environ["SHELL"] = "/bin/bash"
-    current_dir = Path(__file__).resolve()
-    project_name = 'docs'
-    root_dir = next(
-        p for p in current_dir.parents if p.parts[-1] == project_name
-    )
+    root_dir = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
I changed this because the `available_software.py` script was failing in a checkout [my fork](https://github.com/boegel/EESSI-docs), because the top-level directory there was `EESSI-docs` rather than `docs`.

I don't see the need to use `pathlib` here, it makes the code a whole lot more messy and hard to understand, imho